app/Http/Controllers/Backend/Helper/StatusHelper.php

```diff
@@ -66,9 +66,9 @@ private function generateSocialText(): string {
             $body           = $this->mastodon ? $this->getMastodonBody() : $this->status->body;
             $appendix       = $this->generateAppendix();
             $appendixLength = strlen($appendix) + 30;
-            $postText       = substr($body, 0, 280 - $appendixLength);
+            $postText       = substr($body, 0, 500 - $appendixLength);
```

Comment:
This change would not change anything, as the status body cannot be longer than 280 characters.